nntrainer/src/neuralnet.cpp

```diff
@@ -794,6 +806,8 @@ int NeuralNetwork::train_run() {
         break;
       }
     }
+
+    saveModel();
```

Comment:
how about save the model if the accuracy is better than previous iteration? May be later this PR?

Reply:
That's a good way. How about adding it as an option than doing it always as some users might want the latest models for inspection etc?
I will add it another PR.

Reply:
That's good. Thanks.